mono/utils/mono-error.c

```diff
@@ -588,6 +599,9 @@ mono_error_prepare_exception (MonoError *oerror, MonoError *error_out)
 	case MONO_ERROR_MISSING_FIELD:
 		exception = mono_corlib_exception_new_with_args ("System", "MissingFieldException", error->full_message, error->first_argument, error_out);
 		break;
+	case MONO_ERROR_MEMBER_ACCESS:
+		exception = mono_corlib_exception_new_with_args ("System", "MemberAccessException", error->full_message, error->first_argument, error_out);
```

Comment:
I think this should be `mono_exception_from_name_msg (mono_defaults.corlib, "System", "MemberAccessException", error->full_message)`.  There's no `MemberAccessException(String,String)` constructor, afaict.